captum/attr/_core/deep_lift.py

```diff
@@ -322,7 +328,7 @@ def attribute(  # type: ignore
                activations. The hooks and attributes will be removed
             after the attribution is finished"""
         )
-
+        self.eps = eps
```

Comment:
Thank you for the pr, @bilalsal ! We usually don't set any state variables in the attribute computation method. We can either pass this in the constructor or pass to the method as an argument instead of making it an instance variable.

Reply:
Thanks for the hint Narine! Yes, the constructor is the safer place (this also reduces the complexity of using .attribute(), it is unlikely users want to use different eps for the same model).

Reply:
Thank you, @bilalsal ! Looks good. There is a small formatting issue. That you can probably also reproduce with `ufmt format .` Let's address that before merging.

Reply:
cc: @bilalsal